GliaWidgets/Public/Glia/Glia.OpaqueAuthentication.swift

```diff
@@ -67,7 +70,8 @@ extension Glia {
                     callback(result.mapError(Glia.Authentication.Error.init) )
                 }
             },
-            deauthenticateWithCallback: { callback in
+            deauthenticateWithCallback: { [weak self] callback in
+                self?.loggerPhase.logger.prefixed(Self.self).info("De-authenticate")
```

Comment:
would it make sense to move this log inside of closure and log result of deauthentication?

Reply:
It makes sense. But ss stated [here](https://github.com/salemove/ios-sdk-widgets/pull/852#issuecomment-1854455410) proposed change has to be done on both platforms and can not be a part of this PR.